app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/SessionsFragment.kt

```diff
@@ -54,6 +56,16 @@ class SessionsFragment : Fragment(), Injectable {
                 }
             }
         })
+        sessionsViewModel.refreshResult.observe(this, { result ->
+            when (result) {
+                is Result.Failure -> {
+                    // If user is offline, not error. So we write log to debug
+                    Timber.d(result.e)
+                    Snackbar.make(view, R.string.session_fetch_failed, Snackbar.LENGTH_SHORT).show()
```

Comment:
It's not the scope of this PR but it might be better to reload button. What do you think?